Benzy-Clubbers/benzy-clubbers-ember
# Added filter by trim and sort button for the list of reviews

What I Did:
- Added Filters bar above the list of reviews. This includes:
  - Filter by trim
  - Sort by rating
  - Sort by content length (review itself)

How to Test:
- Run `npm i` to install new dependencies for material icons
- Go to the car model that has many test reviews. You can also add reviews to test
- Apply filter by trim and check if it properly filters it
- Apply sort by rating or content length. Note that these two are independent and you can't apply two sorts at the same time (there is no reason for that). Test of sorts work with the applied filter
- Test responsiveness of all buttons and elements. They must stack into two lines after the screen gets smaller. The stack of filters and sorts must align with all other boxes around
